model/registry/cicd.yaml

```diff
@@ -0,0 +1,198 @@
+groups:
+  - id: registry.scm
+    prefix: scm
+    type: attribute_group
+    brief: >
+     This group defines the attributes for source control management (SCM) systems.
+  - id: registry.scm.repository
+    prefix: repository
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to repositories within a source control management system.
+    attributes:
+      - id: name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the repository.
+        examples: ['opentelemetry-collector-contrib']
+      - id: ref.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the ref (branch) in the repository.
+        examples: ['my-feature-branch']
+      - id: ref.commit.sha
+        type: string
+        stability: experimental
+        brief: >
+          The sha of the commit the ref (branch) is pointing to in the repository.
+        examples: ['9d59409acf479dfa0df1aa568182e43e43df8bbe28d60fcf2bc52e30068802cc']
+      - id: change.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the change (pull request / merge request) in the repository.
+        examples: ['pull-request-123']
+      - id: change.id
+        type: string
+        stability: experimental
+        brief: >
+          The id of the change (pull request / merge request) in the repository.
+        examples: ['123']
+  - id: registry.incident
```

Comment:
if you define here completely new namespace please create a completely new yaml file for it.
if they all belong to cicd it might make sense to add to all groups `cicd` prefix i.e. `registry.cicd.incident` etc

Reply:
It is a good point that having a namespace clearly identifies the realm of the attributes and avoids confusing them with other realms.

Reply:
[Related Response](https://github.com/open-telemetry/semantic-conventions/pull/1075#discussion_r1621696735)